packages/community-cli-plugin/package.json

```diff
@@ -22,6 +22,7 @@
     "dist"
   ],
   "dependencies": {
+    "@react-native/dev-middleware": "^0.73.0",
     "@react-native-community/cli-server-api": "12.0.0-alpha.9",
     "@react-native-community/cli-tools": "12.0.0-alpha.9",
```

Comment:
do we plan to eventually move these entirely or to some extent to the community-cli-plugin?

Reply:
I'd say we slowly are.
- This is started in https://github.com/facebook/react-native/pull/39078 where I'm trying to decouple a little from `cli-server-api`. (Though no short term plans to fully decouple).
- And for `cli-server-api` we're looking to slowly migrate things into `@react-native/dev-middleware` — we'll probably do this _after_ the 0.73 JSC Remote Debugging deprecation.

⬇️ Apologies for the amount of merge attempts here, trying to line this up with our internal CI and infra 😅. Landing soon!